scoring.js

```diff
@@ -0,0 +1,90 @@
+const utils = require('./utilities')
+// import * as utils from './utilities'
+
+const calculateScore = (player) => {
+
+    var availablePositions = ['QB', 'RB', 'TE', 'WR'];
+    var currentPosition = player.position;
+    if (!availablePositions.includes(currentPosition)){
```

Comment:
we went over this but I am going to still give you feedback.  Break off into smaller functions and do a switch statement